### PR TITLE
Security Armor Update

### PR DESCRIPTION
**Pull request by DaveTheHeadcrab**
Re-Icons The Security Helmet:
newhelmet

Adds a new security hat type, as suggested on the forums. It can be found in the wardrobe:
newhat

Re-icons the security vests to look more like vests rather than armor plates. You can now attach holobadges to sec vests.
newarmor

secarmor help
newarmor hat

Helmet and Vest icons (Minus the holobadge bit) are made by /tg/
The officer cap is a holdover from the long-dead customs officer, RIP in piss.

### CODE REVIEW

**Comment by DaveTheHeadcrab** [comment]
MORE ROGUE COMMITS. GOD, WHY?

**Comment by monster860** [comment]
You're just that *not good* at git.

**Comment by Spacemanspark** [comment]
I honestly prefer the old armor sprite.

**Comment by pipextiny1** [comment]
New new security hat type looks SS to me +1

**Comment by Regen1** [comment]
Helmet shouldn't have glasses on them, looks like an aviator helmet

**Comment by FlattestGuitar** [comment]
I love everything but the armor.

I think this one looks worse than the current ones.

/opinion

**Comment by DaveTheHeadcrab** [comment]
The helmet has the visior like our current helmet does.
You can see secHUDs under the visor in that picture, though.

**Comment by Ar3nn** [comment]
It all looks pretty good to me. :+1:

**Comment by taukausanake** [comment]
I like the sprites we already have, and that's not me being purist here. These sprites just don't do it for me. The helmet and vest look a little off. The hat is alright but can't say I care for it too much either

**Comment by Fox-McCloud** [comment]
I like the cap, for sure.

**Comment by TullyBurnalot** [comment]
Pretteh. :+1:

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -36,9 +36,43 @@
...
 /obj/item/clothing/suit/armor/vest/security
 	name = "security armor"
-	desc = "An armored vest that protects against some damage. This one has Nanotrasen corporate badge."
-	icon_state = "armorsec"
+	desc = "An armored vest that protects against some damage. This one has a clip for a holobadge."
+	icon_state = "armor"
 	item_state = "armor"
+	var/obj/item/clothing/accessory/holobadge/attached_badge
+
+/obj/item/clothing/suit/armor/vest/security/attackby(obj/item/W as obj, mob/user as mob, params)
```
Comment (Fox-McCloud) [inline code/modules/clothing/suits/armor.dm]: This should all be handled on the holobadget accessory side of things, like collars, ties, scarves, and the likes

Comment (Fox-McCloud) [inline code/modules/clothing/suits/armor.dm]: I realized I'm an idiot and that this is, in fact, a suit and not uniform. Yeah, this is the best way to handle it.

**Comment by FlattestGuitar** [comment]
Hey, I don't want to be *that guy*, but most comments here say that the new armor sprites aren't as good as the current ones. Shouldn't we try to find better ones before this hits the code?

**Comment by Spacemanspark** [comment]
Too late

**Pull request merged**